src/main/resources/application-docker.yaml

```diff
@@ -0,0 +1,16 @@
+spring:
+  autoconfigure:
+    exclude: org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
+  data:
+    mongodb:
+      uri: mongodb://mongo:27017/telegram-bot-db
+bot:
+  token: 6565348709:AAES0cSZrxrspwIDy6zzVhnCtRwMWveoYYc
```

Comment:
Never push tokens from third-party resources to the github, as there are bots that scan all open repositories and steal them

Reply:
My bad, i forgot to hide it in the new profile